test/lib/utils/group-helper.test.js

```diff
@@ -0,0 +1,52 @@
+const { parse, sep } = require('path');
+const GroupHelper = require('../../../lib/utils/group-helper');
+
+describe('group-helper tests', () => {
+    describe('resolveFilePath', () => {
+        it('should return correct path : sample/entry.js', done => {
+            const GroupHelperOptions = [{ entry: 'test/lib/utils/sample/entry.js' }, { progress: 'bar' }, { sourcemap: 'eval' }, { dev: false }, { prod: true }];
+            const GroupHelperObj = new GroupHelper(GroupHelperOptions);
+            const entryPath = GroupHelperObj.resolveFilePath(GroupHelperOptions[0].entry);
+            const { dir, base } = parse(entryPath);
+            const entryOutput = dir.split(sep).concat(base);
+            expect(entryOutput.join('/').includes(GroupHelperOptions[0].entry)).toBe(true);
```

Comment:
I think this functionality also would need a better abstraction

Reply:
I will try to refactore it.

Reply:
Awesome!